spec/rails-5.1.1/.gitignore

```diff
@@ -0,0 +1 @@
+/log/*.log
```

Comment:
`rails-5.1.1` directory was aligned with naming of `rails-4.2.7` directory. I'll also rename that to `rails-4.2` then.